man/man8/zfs.8

```diff
@@ -4417,11 +4417,12 @@ The program can be used to gather information such as properties and
 determining if changes would succeed (zfs.check.*).
 Without this flag, all pending changes must be synced to disk before
 a channel program can complete.
-.It Fl t Ar timeout
-Execution time limit, in milliseconds.
-If a channel program executes for longer than the provided timeout, it will
-be stopped and an error will be returned.
-The default timeout is 1000 ms, and can be set to a maximum of 10000 ms.
+.It Fl t Ar instruction-limit
+Execution time limit, in number of Lua instructions to execute.
```

Comment:
This still says time limit, should likely read:

Limit the number of Lua instructions to be executed.

Reply:
Sure, fixed it here and in zfs-program.8